src/ig/IGExporter.ts

```diff
@@ -61,27 +58,16 @@ const CONFORMANCE_AND_TERMINOLOGY_RESOURCES = new Set([
 export class IGExporter {
   private ig: ImplementationGuide;
   private readonly configPath: string;
-  private readonly outputLog: Map<string, outputLogDetails>;
   private readonly config: Configuration;
   private readonly configName: string;
-  private shouldCopyFiles: boolean;
   constructor(
     private readonly pkg: Package,
     private readonly fhirDefs: FHIRDefinitions,
-    private readonly igDataPath: string,
-    private readonly isIgPubContext = false
+    private readonly igDataPath: string
```

Comment:
Correct. This no longer has anything to do with `ig-data` the folder. It will be the location where the other data for the IG lives, which is why I didn't change the name. But I do think this is really just the input path, so I'll update it!

Reply:
`igData` is gone, but IG data is still with us. f8706fe